### PR TITLE
add `tpl_args` for `write` subcommand

### PR DESCRIPTION
**Pull request by illfygli**
Added this in my fork so that `himalaya write` can be used as a `mailto:`handler.

Happy to make any changes needed for merge if you think this addition makes sense.

### CODE REVIEW

**Comment by soywod** [comment]
> Added this in my fork so that `himalaya write` can be used as a
> `mailto:`handler.

Thanks a lot for your contribution! I thought about this not so long
time ago and it is a very good addition.

> Happy to make any changes needed for merge if you think this addition
> makes sense.

I see that you added the `--to` arg, but in fact we could add other args
like `--cc`, `--subject` etc. Have a look at the `TplOverride` struct in
`tpl_args`, it does exactly this.

**Comment by illfygli** [comment]
Thanks for the hint, I gave it a go. :)

File: cli/src/msg/msg_entity.rs [hunk 1]
```diff
@@ -425,6 +426,11 @@ impl Msg {
         self
     }
 
+    pub fn to(mut self, to: Vec<&str>) -> Result<Self> {
```
Comment (soywod) [inline cli/src/msg/msg_entity.rs:429]: This function can be removed if it is not used any more.

**Review by soywod — CHANGES_REQUESTED** [review]
Sounds really great! I just have a last comment about dead code, and we can merge it.

**Comment by soywod** [comment]
I also noticed that you targeted the wrong branch (`master`), I just switched it to `development`.

**Comment by illfygli** [comment]
Now I discovered that it already works as a mailto-handler with just `himalaya` and no subcommand. :smile: 

But good to have this option as well, I think.

**Comment by soywod** [comment]
> Now I discovered that it already works as a mailto-handler with just
> `himalaya` and no subcommand.

Ah yes indeed :) I realized that the mailto support is not documented at
all. I will update the README.

> But good to have this option as well, I think.

Yes anyway. Thanks a lot!

**Pull request merged**